hastexo/static/html/main.html

```diff
@@ -1,4 +1,5 @@
-<div class="hastexo_block">
+{{child_content|safe}}
+<div class="hastexblock">
```

Comment:
No.  The only reference to this class is in main.js.  I had to change it because JQuery refused to find it, due to the underscore.  More likely than not a JQuery bug, but I figured changing this was the more efficient fix.

Reply:
OK, fair enough. :)

Reply:
It would be good if you could mention this in the commit message though. Or in an inline comment, lest a future adopter change this to something that _does_ involve an underscore again, mysteriously breaking things.